vyxal/elements.py

```diff
@@ -2854,9 +2856,24 @@ def regex_sub(lhs, rhs, other, ctx):
 def remove(lhs, rhs, ctx):
     """Element o
     (any, any) -> a.remove(b)
+    (fun, num) -> first b positive integers where a is truthy
     """
     lhs = iterable(lhs)
     ts = vy_type(lhs)
+    if set(vy_type(lhs, rhs)) == {types.FunctionType, NUMBER_TYPE}:
+        lhs, rhs = (rhs, lhs) if ts is types.FunctionType else (lhs, rhs)
```

Comment:
We really need a better way to check types and swap than this :\. Not a big deal though, you can resolve this comment and we can work on it later